.circleci/config.yml

```diff
@@ -0,0 +1,27 @@
+version: 2
```

Comment:
remove this after proof of concept.  we'll be building directly on Jenkins instead